test/jest/Corporation.test.ts

```diff
@@ -117,58 +117,24 @@ describe("Corporation", () => {
     });
   });
 
-  describe("helpers.calculateOfficeSizeUpgradeCost", () => {
-    // These values were pulled from v2.6.0 as reference test values
-    const refCosts = {
-      from3: {
-        3: 4360000000,
-        15: 26093338259.60001,
-        150: 3553764305895.2593,
+  describe("helpers.calculateOfficeSizeUpgradeCost matches documented formula", () => {
+    // for discussion and computation of these test values, see:
+    // https://github.com/bitburner-official/bitburner-src/pull/1179#discussion_r1534948725
+    it.each([
+      { fromSize: 3, increaseBy: 3, expectedCost: 4360000000.0 },
+      { fromSize: 3, increaseBy: 15, expectedCost: 26093338259.6 },
+      { fromSize: 3, increaseBy: 150, expectedCost: 3553764305895.24902 },
+      { fromSize: 6, increaseBy: 3, expectedCost: 4752400000.0 },
+      { fromSize: 6, increaseBy: 15, expectedCost: 28441738702.964 },
+      { fromSize: 6, increaseBy: 150, expectedCost: 3873603093425.821 },
+      { fromSize: 9, increaseBy: 3, expectedCost: 5180116000.0 },
+      { fromSize: 9, increaseBy: 15, expectedCost: 31001495186.23076 },
+      { fromSize: 9, increaseBy: 150, expectedCost: 4222227371834.145 },
+    ])(
+      "should cost $expectedCost to upgrade office by $increaseBy from size $fromSize",
+      ({ fromSize, increaseBy, expectedCost }) => {
+        expect(calculateOfficeSizeUpgradeCost(fromSize, increaseBy as PositiveInteger)).toBeCloseTo(expectedCost, 1);
```

Comment:
Can you take out the `1` now? Although I wouldn't be surprised if not, Jests implementation of `toBeCloseTo` is completely dumb (it's strictly absolute error instead of relative error).

Reply:
I can't, unfortunately.  I left it out at first when I moved to the new testing style, but the default 2 is too stringent and the +150 tests all failed.